helpers/authentication.js

```diff
@@ -69,6 +69,10 @@ const populateCurrentUser = (req, res) => {
 				res.locals.currentSchoolData.isExpertSchool = data2.purpose === 'expert';
 				return data2;
 			});
+		}).catch((e) => {
+			if (e.error.message === 'jwt expired' || e.error.className === 'not-found') {
+				res.clearCookie('jwt');
```

Comment:
there is much more to do on logout, maybe redirect instead and apply the regular logout routine again or is that not working with an jwt that is outdated?

Reply:
You can't logout without valid jwt